openreview/stages/venue_stages.py

```diff
@@ -293,6 +295,20 @@ def get_content(self, api_version='1', conference=None, venue_id=None):
                     }
                 }
 
+            if self.commitments_venue:
```

Comment:
I would check if paper_link is already in the content and don't overwrite it, the PCs might want to change the description.